### PR TITLE
[Cookbook][Doctrine] some tweaks to the Doctrine registration article

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

**Comment by WouterJ** [comment]
:+1:

**Comment by javiereguiluz** [comment]
:+1:

File: cookbook/doctrine/registration_form.rst [hunk 1]
```diff
@@ -213,10 +215,9 @@ into the database::
     // src/AppBundle/Controller/RegistrationController.php
     namespace AppBundle\Controller;
 
-    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
-
     use AppBundle\Form\UserType;
     use AppBundle\Entity\User;
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
     use Symfony\Component\HttpFoundation\Request;
     use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
```
Comment (HeahDude) [inline cookbook/doctrine/registration_form.rst]: "Sensio" could go before "Symfony" to keep the alphabetical order.

Same for [line 46](#diff-3efb51c932d0635479c8d3bafb824c7fR46) can be the last.

[Line 250](#diff-3efb51c932d0635479c8d3bafb824c7fR250) "[...] like send**ing** [...] ?
[Line 414](#diff-3efb51c932d0635479c8d3bafb824c7fR414) shouldn't it be "[...] Symfony knows __how__ to [...]"

Comment (xabbuh) [inline cookbook/doctrine/registration_form.rst]: thanks, I have made these tweaks in 44d747987b3fbbd1ad791702365b1dbcae2d2e29

Comment (HeahDude) [inline cookbook/doctrine/registration_form.rst]: :+1: I was not sure I should mention it

**Pull request merged**